### DISCUSSION TITLE
Improve guidance about how to use Jupyter notebooks in a workflow

### DISCUSSION THREAD

**Discussion by adrn** [rodluger/showyourwork#676] [Q&A]
Currently, Jupyter is only mentioned here: https://show-your.work/en/latest/config/?highlight=notebook#scripts

But probably also relevant to mention https://github.com/mwouts/jupytext and maybe add some descriptive examples.

**Comment by HealthyPear** [comment]
Regarding how to use notebooks what is in the docs might be limited as there are multiple ways to do that.

What I describe in #448 should be a rather standard and modern way to approach the problem.

Take also into account that notebooks [can be integrated natively into any snakemake workflow](https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#jupyter-notebook-integration).

About jupytext I have an idea of at least adding a basic config file to the cookiecutter template so it should be ready to use, but I also want to limit as much as possible to push on the user any config choice from showyourwork, so we could also add this to showyourwork.yml and let the workflow update the file for the user as he/she pleases.
